py-publishing.Rmd

```diff
@@ -0,0 +1,487 @@
+# Publishing {#py-publishing}
+
+## Questions {#py-publishing-questions}
+
+```{r, child="questions/py-publishing.md"}
+```
+
+## Why should I share my work on the internet?
+
+A key part of any project is communicating what you have learned.
+You've already seen how to create documents to communicate your work with R Markdown,
```

Comment:
R Markdown? Replace with python equivalent, is it Jupyter Notebooks in this instance?

Reply:
Hmmm, I thought I replaced all these, apparently I missed three!